runtime/compiler/runtime/JITServerAOTCache.cpp

```diff
@@ -867,7 +874,7 @@ JITServerAOTCache::getClassRecord(const AOTCacheClassLoaderRecord *classLoaderRe
    if (!JITServerAOTCacheMap::cacheHasSpace())
       return NULL;
 
-   auto record = AOTCacheClassRecord::create(_nextClassId, classLoaderRecord, hash, size,
+   auto record = AOTCacheClassRecord::create(_nextClassId, classLoaderRecord, hash, romClass->romSize,
```

Comment:
Here I use the `romClass->romSize` of the original packed ROM class always. Before, this would be the `romSize` of the repacked `romClass` if `romClass` were runtime-generated. I think this is okay - the repacked `romClass` differs from the original packed `romClass` only in that the occurrences of the name of the `romClass` in itself get truncated to the "determinstic class name prefix".

Reply:
ok. Maybe add a comment to that extent.